code/game/objects/items/stacks/stack.dm

```diff
@@ -204,7 +204,6 @@
 			var/datum/matter_synth/S = synths[i]
 			S.use_charge(charge_costs[i] * used) // Doesn't need to be deleted
 		return 1
-	return 0
```

Comment:
whats wrong with a trailing return 0? isnt the default return value null?

Reply:
The problem here is that this coding line could never be reached. 
above this was:
`
if(something)
  code
  return
else
   code
   return
return
`